<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>ID</th>
      <th>SCORE</th>
      <th>CHROM</th>
      <th>START</th>
      <th>END</th>
      <th>OBSERVED</th>
      <th>VCF</th>
      <th>SNPEFF</th>
      <th>DBSNP</th>
      <th>RARE</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>chr17:g.80393843G&gt;A</td>
      <td>1.0</td>
      <td>17</td>
      <td>80393843</td>
      <td>80393843</td>
      <td>True</td>
      <td>alt:A; position:80393843; ref:G;</td>
      <td>effect:intron_variant; feature_id:NM_173620.2; feature_type:transcript; gene_id:HEXDC; genename:HEXDC; hgvs_c:c.631+95G&gt;A; putative_impact:MODIFIER; rank:6; total:11; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'G', 'freq': {'gnomad': 1.0}}, {'allele': 'A', 'freq': {'gnomad': 0.0}}]; alt:A; chrom:17; dbsnp_build:156; gene:{'geneid': 284004, 'is_pseudo': False, 'name': 'hexosaminidase D', 'rnas': [{'hgvs': 'NM_001330542.2:c.631+95=', 'protein_product': {'refseq': 'NP_001317471.1'}, 'refseq': 'NM_001330542.2', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_001369487.1:c.355+95=', 'protein_product': {'refseq': 'NP_001356416.1'}, 'refseq': 'NM_001369487.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_001369488.1:c.355+95=', 'protein_product': {'refseq': 'NP_001356417.1'}, 'refseq': 'NM_001369488.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_173620.3:c.631+95=', 'protein_product': {'refseq': 'NP_775891.2'}, 'refseq': 'NM_173620.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_006722280.3:c.490+95=', 'protein_product': {'refseq': 'XP_006722343.1'}, 'refseq': 'XM_006722280.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_011523560.3:c.523+95=', 'protein_product': {'refseq': 'XP_011521862.1'}, 'refseq': 'XM_011523560.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_017024484.2:c.631+95=', 'protein_product': {'refseq': 'XP_016879973.1'}, 'refseq': 'XM_017024484.2', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_024306473.1'}, 'refseq': 'XM_024450705.2', 'so': [{'accession': 'SO:0002153', 'name': 'genic_upstream_transcript_variant'}]}, {'hgvs': 'XM_047435784.1:c.355+95=', 'protein_product': {'refseq': 'XP_047291740.1'}, 'refseq': 'XM_047435784.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_047435785.1:c.355+95=', 'protein_product': {'refseq': 'XP_047291741.1'}, 'refseq': 'XM_047435785.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_047435786.1:c.631+95=', 'protein_product': {'refseq': 'XP_047291742.1'}, 'refseq': 'XM_047435786.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_047291743.1'}, 'refseq': 'XM_047435787.1', 'so': [{'accession': 'SO:0002153', 'name': 'genic_upstream_transcript_variant'}]}, {'hgvs': 'XM_047435788.1:c.631+95=', 'protein_product': {'refseq': 'XP_047291744.1'}, 'refseq': 'XM_047435788.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'refseq': 'XR_007065292.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'refseq': 'XR_007065293.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}], 'strand': '+', 'symbol': 'HEXD'}; hg19:{'end': 80393843, 'start': 80393843}; ref:G; rsid:rs2053749987; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr17:g.18054389C&gt;T</td>
      <td>1.0</td>
      <td>17</td>
      <td>18054389</td>
      <td>18054389</td>
      <td>True</td>
      <td>alt:T; position:18054389; ref:C;</td>
      <td>effect:intron_variant; feature_id:NM_016239.3; feature_type:transcript; gene_id:MYO15A; genename:MYO15A; hgvs_c:c.7474-35C&gt;T; putative_impact:MODIFIER; rank:37; total:64; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'C', 'freq': {'exac': 1.0, 'gnomad_exomes': 1.0}}, {'allele': 'T', 'freq': {'exac': 0.0, 'gnomad_exomes': 0.0}}]; alt:T; chrom:17; dbsnp_build:156; gene:[{'geneid': 51168, 'is_pseudo': False, 'name': 'myosin XVA', 'rnas': [{'hgvs': 'NM_016239.4:c.7474-35=', 'protein_product': {'refseq': 'NP_057323.3'}, 'refseq': 'NM_016239.4', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_011522220.1'}, 'refseq': 'XM_011523918.3', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'hgvs': 'XM_017024714.3:c.7414-35=', 'protein_product': {'refseq': 'XP_016880203.1'}, 'refseq': 'XM_017024714.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_017024715.3:c.7477-35=', 'protein_product': {'refseq': 'XP_016880204.1'}, 'refseq': 'XM_017024715.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_024306548.1'}, 'refseq': 'XM_024450780.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_024306549.1'}, 'refseq': 'XM_024450781.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'refseq': 'XR_934039.3', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}], 'strand': '+', 'symbol': 'MYO15A'}, {'geneid': 124903944, 'is_pseudo': False, 'name': 'uncharacterized LOC124903944', 'rnas': [{'refseq': 'XR_007065652.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}], 'strand': '-', 'symbol': 'LOC124903944'}]; hg19:{'end': 18054389, 'start': 18054389}; ref:C; rsid:rs765799923; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr17:g.7674149G&gt;A</td>
      <td>1.0</td>
      <td>17</td>
      <td>7674149</td>
      <td>7674149</td>
      <td>True</td>
      <td>alt:A; position:7674149; ref:G;</td>
      <td>cdna:{'length': '14955', 'position': '5720'}; cds:{'length': '13284', 'position': '4260'}; effect:synonymous_variant; feature_id:NM_020877.3; feature_type:transcript; gene_id:DNAH2; genename:DNAH2; hgvs_c:c.4260G&gt;A; hgvs_p:p.Lys1420Lys; protein:{'length': '4427', 'position': '1420'}; putative_impact:LOW; rank:27; total:86; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'G', 'freq': {'dbgap_popfreq': 1.0, 'exac': 1.0, 'topmed': 1.0}}, {'allele': 'A', 'freq': {'dbgap_popfreq': 0.0, 'exac': 0.0, 'topmed': 0.0}}]; alt:A; chrom:17; dbsnp_build:156; gene:{'geneid': 146754, 'is_pseudo': False, 'name': 'dynein axonemal heavy chain 2', 'rnas': [{'protein_product': {'refseq': 'NP_001290199.1'}, 'refseq': 'NM_001303270.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 5327, 'seq_id': 'NM_020877.5'}, 'hgvs': 'NM_020877.5:c.4260=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1419, 'seq_id': 'NP_065928.2'}}}, 'protein_product': {'refseq': 'NP_065928.2'}, 'refseq': 'NM_020877.5', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_011523663.2'}, 'hgvs': 'XM_011523663.2:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_011521965.1'}}}, 'protein_product': {'refseq': 'XP_011521965.1'}, 'refseq': 'XM_011523663.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4495, 'seq_id': 'XM_011523667.3'}, 'hgvs': 'XM_011523667.3:c.4260=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1419, 'seq_id': 'XP_011521969.1'}}}, 'protein_product': {'refseq': 'XP_011521969.1'}, 'refseq': 'XM_011523667.3', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_011523670.3'}, 'hgvs': 'XM_011523670.3:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_011521972.1'}}}, 'protein_product': {'refseq': 'XP_011521972.1'}, 'refseq': 'XM_011523670.3', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_017024219.2'}, 'hgvs': 'XM_017024219.2:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_016879708.1'}}}, 'protein_product': {'refseq': 'XP_016879708.1'}, 'refseq': 'XM_017024219.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4741, 'seq_id': 'XM_047435424.1'}, 'hgvs': 'XM_047435424.1:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_047291380.1'}}}, 'protein_product': {'refseq': 'XP_047291380.1'}, 'refseq': 'XM_047435424.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_047435425.1'}, 'hgvs': 'XM_047435425.1:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_047291381.1'}}}, 'protein_product': {'refseq': 'XP_047291381.1'}, 'refseq': 'XM_047435425.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2967, 'seq_id': 'XM_047435426.1'}, 'hgvs': 'XM_047435426.1:c.2898=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 965, 'seq_id': 'XP_047291382.1'}}}, 'protein_product': {'refseq': 'XP_047291382.1'}, 'refseq': 'XM_047435426.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2670, 'seq_id': 'XM_047435427.1'}, 'hgvs': 'XM_047435427.1:c.2511=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 836, 'seq_id': 'XP_047291383.1'}}}, 'protein_product': {'refseq': 'XP_047291383.1'}, 'refseq': 'XM_047435427.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2653, 'seq_id': 'XM_047435428.1'}, 'hgvs': 'XM_047435428.1:c.2502=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 833, 'seq_id': 'XP_047291384.1'}}}, 'protein_product': {'refseq': 'XP_047291384.1'}, 'refseq': 'XM_047435428.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}], 'strand': '+', 'symbol': 'DNAH2'}; hg19:{'end': 7674149, 'start': 7674149}; ref:G; rsid:rs751380125; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr12:g.94071094A&gt;C</td>
      <td>1.0</td>
      <td>12</td>
      <td>94071094</td>
      <td>94071094</td>
      <td>True</td>
      <td>alt:C; position:94071094; ref:A;</td>
      <td>distance_to_feature:399; effect:upstream_gene_variant; feature_id:NM_001320099.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_001320101.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_003805.4; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_001320100.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NR_135147.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:n.-53A&gt;C; putative_impact:MODIFIER; transcript_biotype:pseudogene; |distance_to_feature:1451; effect:upstream_gene_variant; feature_id:NM_001330126.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |effect:intergenic_region; feature_id:SOCS2-CRADD; feature_type:intergenic_region; gene_id:SOCS2-CRADD; genename:SOCS2-CRADD; hgvs_c:n.94071094A&gt;C; putative_impact:MODIFIER; |</td>
      <td>alleles:[{'allele': 'A', 'freq': {'dbgap_popfreq': 1.0, 'gnomad': 1.0, 'topmed': 1.0}}, {'allele': 'C', 'freq': {'dbgap_popfreq': 0.0, 'gnomad': 0.0, 'topmed': 0.0}}]; alt:C; chrom:12; dbsnp_build:156; gene:{'geneid': 8738, 'is_pseudo': False, 'name': 'CASP2 and RIPK1 domain containing adaptor with death domain', 'rnas': [{'protein_product': {'refseq': 'NP_001307028.1'}, 'refseq': 'NM_001320099.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001307029.1'}, 'refseq': 'NM_001320100.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001307030.1'}, 'refseq': 'NM_001320101.3', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001317055.1'}, 'refseq': 'NM_001330126.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_003796.1'}, 'refseq': 'NM_003805.5', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'refseq': 'NR_135147.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_016875634.1'}, 'refseq': 'XM_017020145.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_024305020.1'}, 'refseq': 'XM_024449252.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_047285760.1'}, 'refseq': 'XM_047429804.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_047285761.1'}, 'refseq': 'XM_047429805.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}], 'so': [{'accession': 'SO:0001636', 'name': '2KB_upstream_variant'}], 'strand': '+', 'symbol': 'CRADD'}; hg19:{'end': 94071094, 'start': 94071094}; ref:A; rsid:rs1173928494; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr5:g.453975C&gt;T</td>
      <td>1.0</td>
      <td>5</td>
      <td>453975</td>
      <td>453975</td>
      <td>True</td>
      <td>alt:T; position:453975; ref:C;</td>
      <td>cdna:{'length': '2762', 'position': '983'}; cds:{'length': '2238', 'position': '855'}; effect:synonymous_variant; feature_id:NM_007277.4; feature_type:transcript; gene_id:EXOC3; genename:EXOC3; hgvs_c:c.855C&gt;T; hgvs_p:p.Tyr285Tyr; protein:{'length': '745', 'position': '285'}; putative_impact:LOW; rank:4; total:13; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'C', 'freq': {'dbgap_popfreq': 1.0, 'exac': 1.0, 'gnomad': 1.0, 'gnomad_exomes': 1.0, 'goesp': 1.0, 'mgp': 0.998, 'topmed': 1.0}}, {'allele': 'T', 'freq': {'dbgap_popfreq': 0.0, 'exac': 0.0, 'gnomad': 0.0, 'gnomad_exomes': 0.0, 'goesp': 0.0, 'mgp': 0.002, 'topmed': 0.0}}]; alt:T; chrom:5; dbsnp_build:156; gene:{'geneid': 11336, 'is_pseudo': False, 'name': 'exocyst complex component 3', 'rnas': [{'codon_aligned_transcript_change': {'deleted_sequence': 'TAC', 'inserted_sequence': 'TAC', 'position': 1023, 'seq_id': 'NM_007277.5'}, 'hgvs': 'NM_007277.5:c.855=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'Y', 'inserted_sequence': 'Y', 'position': 284, 'seq_id': 'NP_009208.2'}}}, 'protein_product': {'refseq': 'NP_009208.2'}, 'refseq': 'NM_007277.5', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'TAC', 'inserted_sequence': 'TAC', 'position': 1023, 'seq_id': 'XM_047416683.1'}, 'hgvs': 'XM_047416683.1:c.855=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'Y', 'inserted_sequence': 'Y', 'position': 284, 'seq_id': 'XP_047272639.1'}}}, 'protein_product': {'refseq': 'XP_047272639.1'}, 'refseq': 'XM_047416683.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}], 'strand': '+', 'symbol': 'EXOC3'}; hg19:{'end': 453975, 'start': 453975}; ref:C; rsid:rs376240114; vartype:snv;</td>
      <td>+</td>
    </tr>
  </tbody>
</table>