<!DOCTYPE html>
<html lang="pl">
<head>
    <meta charset="UTF-8">
    <title>Projekt</title>
    <link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
    <style>
        body {{
            text-align: center;
            font-family: Comic Sans MS;
            background-image: url('https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExMzZ0czE4eW4wZWsxNWhiaGh3cjhoZHFxOW9leWd5eDZ2cGZvbnczdCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ECWLeGxcKasCc/giphy.gif');
            background-size: cover;
            color: #000;
        }}
        table.display {{
            background-color: rgba(255,255,255,0.8);
            margin: 20px auto;
            border-collapse: collapse;
        }}
        th, td {{ padding: 10px; border: 1px solid #aaa; }}
    </style>
    <script src="https://code.jquery.com/jquery-3.7.0.js"></script>
    <script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
</head>
<body>
    <h1>PROJEKTTTT</h1>
    <img src="https://media.giphy.com/media/13borq7Zo2kulO/giphy.gif" width="300" alt="Jednorożec">
    <img src="https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExY245NG84OXE5eDk1ZW5mOG1xM3E5bHR6emR6OHRjNmtxNHViajdhZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/2A75RyXVzzSI2bx4Gj/giphy.gif" width="300" alt="Tęcza">

    <div id="wynik"><table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>ID</th>
      <th>SCORE</th>
      <th>CHROM</th>
      <th>START</th>
      <th>END</th>
      <th>OBSERVED</th>
      <th>VCF</th>
      <th>CLINVAR</th>
      <th>SNPEFF</th>
      <th>DBSNP</th>
      <th>RARE</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>chr16:g.28883241A&gt;G</td>
      <td>1.0</td>
      <td>16</td>
      <td>28883241</td>
      <td>28883241</td>
      <td>True</td>
      <td>alt:G; position:28883241; ref:A;</td>
      <td>allele_id:1223281; alt:G; chrom:16; cytogenic:16p11.2; gene:{'id': '25970', 'symbol': 'SH2B1'}; hg19:{'end': 28883241, 'start': 28883241}; hg38:{'end': 28871920, 'start': 28871920}; hgvs:{'coding': ['NM_001145795.2:c.1450A&gt;G', 'NM_001145796.2:c.1450A&gt;G', 'NM_001145797.2:c.1450A&gt;G', 'NM_001145812.2:c.1450A&gt;G', 'NM_001308293.2:c.1450A&gt;G', 'NM_001308294.2:c.442A&gt;G', 'NM_001387404.1:c.1450A&gt;G', 'NM_001387430.1:c.1450A&gt;G', 'NM_015503.3:c.1450A&gt;G'], 'genomic': ['NC_000016.10:g.28871920A&gt;G', 'NC_000016.9:g.28883241A&gt;G', 'NG_029706.2:g.30321A&gt;G'], 'protein': ['NP_001139267.1:p.Thr484Ala', 'NP_001139268.1:p.Thr484Ala', 'NP_001139269.1:p.Thr484Ala', 'NP_001139284.1:p.Thr484Ala', 'NP_001295222.1:p.Thr484Ala', 'NP_001295223.1:p.Thr148Ala', 'NP_001374333.1:p.Thr484Ala', 'NP_001374359.1:p.Thr484Ala', 'NP_056318.2:p.Thr484Ala']}; rcv:{'accession': 'RCV001621215', 'clinical_significance': 'Benign', 'conditions': {'identifiers': {'medgen': 'C3661900'}, 'name': 'not provided', 'synonyms': ['none provided']}, 'last_evaluated': '2024-01-31', 'number_submitters': 3, 'origin': 'germline', 'preferred_name': 'NM_001387430.1(SH2B1):c.1450A&gt;G (p.Thr484Ala)', 'review_status': 'criteria provided, multiple submitters, no conflicts'}; ref:A; rsid:rs7498665; type:single nucleotide variant; variant_id:1232867;</td>
      <td>cdna:{'length': '2936', 'position': '1722'}; cds:{'length': '2271', 'position': '1450'}; effect:missense_variant; feature_id:NM_001145795.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '756', 'position': '484'}; putative_impact:MODERATE; rank:6; total:9; transcript_biotype:protein_coding; |cdna:{'length': '3192', 'position': '1978'}; cds:{'length': '2271', 'position': '1450'}; effect:missense_variant; feature_id:NM_001308293.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '756', 'position': '484'}; putative_impact:MODERATE; rank:8; total:11; transcript_biotype:protein_coding; |cdna:{'length': '3036', 'position': '1722'}; cds:{'length': '2016', 'position': '1450'}; effect:missense_variant; feature_id:NM_001145796.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '671', 'position': '484'}; putative_impact:MODERATE; rank:6; total:10; transcript_biotype:protein_coding; |cdna:{'length': '3089', 'position': '1722'}; cds:{'length': '2052', 'position': '1450'}; effect:missense_variant; feature_id:NM_001145797.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '683', 'position': '484'}; putative_impact:MODERATE; rank:6; total:10; transcript_biotype:protein_coding; |cdna:{'length': '1971', 'position': '657'}; cds:{'length': '1008', 'position': '442'}; effect:missense_variant; feature_id:NM_001308294.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.442A&gt;G; hgvs_p:p.Thr148Ala; protein:{'length': '335', 'position': '148'}; putative_impact:MODERATE; rank:5; total:9; transcript_biotype:protein_coding; |cdna:{'length': '3135', 'position': '1821'}; cds:{'length': '2016', 'position': '1450'}; effect:missense_variant; feature_id:NM_001145812.1; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '671', 'position': '484'}; putative_impact:MODERATE; rank:6; total:10; transcript_biotype:protein_coding; |cdna:{'length': '4866', 'position': '3552'}; cds:{'length': '2016', 'position': '1450'}; effect:missense_variant; feature_id:NM_015503.2; feature_type:transcript; gene_id:SH2B1; genename:SH2B1; hgvs_c:c.1450A&gt;G; hgvs_p:p.Thr484Ala; protein:{'length': '671', 'position': '484'}; putative_impact:MODERATE; rank:5; total:9; transcript_biotype:protein_coding; |</td>
      <td>alleles:[{'allele': 'A', 'freq': {'1000g': 0.733, 'alspac': 0.599, 'dbgap_popfreq': 0.628, 'estonian': 0.587, 'finrisk': 0.579, 'genome_dk': 0.525, 'gnomad': 0.649, 'gnomad_exomes': 0.649, 'goesp': 0.656, 'gonl': 0.611, 'hapmap': 0.78, 'hgdp_stanford': 0.661, 'korea1k': 0.893, 'korean': 0.873, 'mgp': 0.742, 'northernsweden': 0.578, 'prjeb36033': 0.562, 'qatari': 0.829, 'sgdp_prj': 0.298, 'siberian': 0.382, 'tommo': 0.856, 'topmed': 0.661, 'twinsuk': 0.604}}, {'allele': 'G', 'freq': {'1000g': 0.267, 'alspac': 0.401, 'dbgap_popfreq': 0.372, 'estonian': 0.413, 'finrisk': 0.421, 'genome_dk': 0.475, 'gnomad': 0.351, 'gnomad_exomes': 0.351, 'goesp': 0.344, 'gonl': 0.389, 'hapmap': 0.22, 'hgdp_stanford': 0.339, 'korea1k': 0.107, 'korean': 0.127, 'mgp': 0.258, 'northernsweden': 0.422, 'prjeb36033': 0.438, 'qatari': 0.171, 'sgdp_prj': 0.702, 'siberian': 0.618, 'tommo': 0.144, 'topmed': 0.339, 'twinsuk': 0.396}}, {'allele': 'T', 'freq': {'dbgap_popfreq': 0.0}}]; alt:G; chrom:16; citations:[17228025, 19079261, 19164386, 19478790, 19526209, 19553259, 19910641, 19910938, 20110568, 20127379, 20369022, 20386550, 20397748, 20616199, 20712903, 20725061, 20816152, 20816195, 20824172, 20966902, 21267540, 21283731, 21357383, 21466928, 21750520, 21773002, 21907990, 21912638, 21976109, 22029802, 22051089, 22083549, 22384102, 23270367, 23339409, 23640704, 23712987, 24278702, 24392269, 24528214, 25126397, 25269698, 26075635, 26263223, 26480920, 26495240, 26543368, 26833246, 27465381, 28636007, 28661716, 28977211, 31754094, 31884074, 32228543, 32492003, 33170161, 34556834, 35124268]; dbsnp_build:156; dbsnp_merges:[{'date': '2006-03-11T08:44Z', 'rsid': 'rs17856339', 'rv': 126}, {'date': '2008-05-25T23:56Z', 'rsid': 'rs60209196', 'rv': 130}]; gene:{'geneid': 25970, 'is_pseudo': False, 'name': 'SH2B adaptor protein 1', 'rnas': [{'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1635, 'seq_id': 'NM_001145795.2'}, 'hgvs': 'NM_001145795.2:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001139267.1'}}}, 'protein_product': {'refseq': 'NP_001139267.1'}, 'refseq': 'NM_001145795.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1635, 'seq_id': 'NM_001145796.2'}, 'hgvs': 'NM_001145796.2:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001139268.1'}}}, 'protein_product': {'refseq': 'NP_001139268.1'}, 'refseq': 'NM_001145796.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1635, 'seq_id': 'NM_001145797.2'}, 'hgvs': 'NM_001145797.2:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001139269.1'}}}, 'protein_product': {'refseq': 'NP_001139269.1'}, 'refseq': 'NM_001145797.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1952, 'seq_id': 'NM_001145812.2'}, 'hgvs': 'NM_001145812.2:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001139284.1'}}}, 'protein_product': {'refseq': 'NP_001139284.1'}, 'refseq': 'NM_001145812.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1971, 'seq_id': 'NM_001308293.2'}, 'hgvs': 'NM_001308293.2:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001295222.1'}}}, 'protein_product': {'refseq': 'NP_001295222.1'}, 'refseq': 'NM_001308293.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 570, 'seq_id': 'NM_001308294.2'}, 'hgvs': 'NM_001308294.2:c.442=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 147, 'seq_id': 'NP_001295223.1'}}}, 'protein_product': {'refseq': 'NP_001295223.1'}, 'refseq': 'NM_001308294.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 1841, 'seq_id': 'NM_001387404.1'}, 'hgvs': 'NM_001387404.1:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001374333.1'}}}, 'protein_product': {'refseq': 'NP_001374333.1'}, 'refseq': 'NM_001387404.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 3701, 'seq_id': 'NM_001387430.1'}, 'hgvs': 'NM_001387430.1:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_001374359.1'}}}, 'protein_product': {'refseq': 'NP_001374359.1'}, 'refseq': 'NM_001387430.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'ACA', 'inserted_sequence': 'ACA', 'position': 3701, 'seq_id': 'NM_015503.3'}, 'hgvs': 'NM_015503.3:c.1450=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'T', 'inserted_sequence': 'T', 'position': 483, 'seq_id': 'NP_056318.2'}}}, 'protein_product': {'refseq': 'NP_056318.2'}, 'refseq': 'NM_015503.3', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}], 'strand': '+', 'symbol': 'SH2B1'}; hg19:{'end': 28883241, 'start': 28883241}; ref:A; rsid:rs7498665; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr17:g.80393843G&gt;A</td>
      <td>1.0</td>
      <td>17</td>
      <td>80393843</td>
      <td>80393843</td>
      <td>True</td>
      <td>alt:A; position:80393843; ref:G;</td>
      <td>N/A</td>
      <td>effect:intron_variant; feature_id:NM_173620.2; feature_type:transcript; gene_id:HEXDC; genename:HEXDC; hgvs_c:c.631+95G&gt;A; putative_impact:MODIFIER; rank:6; total:11; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'G', 'freq': {'gnomad': 1.0}}, {'allele': 'A', 'freq': {'gnomad': 0.0}}]; alt:A; chrom:17; dbsnp_build:156; gene:{'geneid': 284004, 'is_pseudo': False, 'name': 'hexosaminidase D', 'rnas': [{'hgvs': 'NM_001330542.2:c.631+95=', 'protein_product': {'refseq': 'NP_001317471.1'}, 'refseq': 'NM_001330542.2', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_001369487.1:c.355+95=', 'protein_product': {'refseq': 'NP_001356416.1'}, 'refseq': 'NM_001369487.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_001369488.1:c.355+95=', 'protein_product': {'refseq': 'NP_001356417.1'}, 'refseq': 'NM_001369488.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'NM_173620.3:c.631+95=', 'protein_product': {'refseq': 'NP_775891.2'}, 'refseq': 'NM_173620.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_006722280.3:c.490+95=', 'protein_product': {'refseq': 'XP_006722343.1'}, 'refseq': 'XM_006722280.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_011523560.3:c.523+95=', 'protein_product': {'refseq': 'XP_011521862.1'}, 'refseq': 'XM_011523560.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_017024484.2:c.631+95=', 'protein_product': {'refseq': 'XP_016879973.1'}, 'refseq': 'XM_017024484.2', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_024306473.1'}, 'refseq': 'XM_024450705.2', 'so': [{'accession': 'SO:0002153', 'name': 'genic_upstream_transcript_variant'}]}, {'hgvs': 'XM_047435784.1:c.355+95=', 'protein_product': {'refseq': 'XP_047291740.1'}, 'refseq': 'XM_047435784.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_047435785.1:c.355+95=', 'protein_product': {'refseq': 'XP_047291741.1'}, 'refseq': 'XM_047435785.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_047435786.1:c.631+95=', 'protein_product': {'refseq': 'XP_047291742.1'}, 'refseq': 'XM_047435786.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_047291743.1'}, 'refseq': 'XM_047435787.1', 'so': [{'accession': 'SO:0002153', 'name': 'genic_upstream_transcript_variant'}]}, {'hgvs': 'XM_047435788.1:c.631+95=', 'protein_product': {'refseq': 'XP_047291744.1'}, 'refseq': 'XM_047435788.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'refseq': 'XR_007065292.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'refseq': 'XR_007065293.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}], 'strand': '+', 'symbol': 'HEXD'}; hg19:{'end': 80393843, 'start': 80393843}; ref:G; rsid:rs2053749987; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr17:g.18054389C&gt;T</td>
      <td>1.0</td>
      <td>17</td>
      <td>18054389</td>
      <td>18054389</td>
      <td>True</td>
      <td>alt:T; position:18054389; ref:C;</td>
      <td>N/A</td>
      <td>effect:intron_variant; feature_id:NM_016239.3; feature_type:transcript; gene_id:MYO15A; genename:MYO15A; hgvs_c:c.7474-35C&gt;T; putative_impact:MODIFIER; rank:37; total:64; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'C', 'freq': {'exac': 1.0, 'gnomad_exomes': 1.0}}, {'allele': 'T', 'freq': {'exac': 0.0, 'gnomad_exomes': 0.0}}]; alt:T; chrom:17; dbsnp_build:156; gene:[{'geneid': 51168, 'is_pseudo': False, 'name': 'myosin XVA', 'rnas': [{'hgvs': 'NM_016239.4:c.7474-35=', 'protein_product': {'refseq': 'NP_057323.3'}, 'refseq': 'NM_016239.4', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_011522220.1'}, 'refseq': 'XM_011523918.3', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'hgvs': 'XM_017024714.3:c.7414-35=', 'protein_product': {'refseq': 'XP_016880203.1'}, 'refseq': 'XM_017024714.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'hgvs': 'XM_017024715.3:c.7477-35=', 'protein_product': {'refseq': 'XP_016880204.1'}, 'refseq': 'XM_017024715.3', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}, {'protein_product': {'refseq': 'XP_024306548.1'}, 'refseq': 'XM_024450780.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_024306549.1'}, 'refseq': 'XM_024450781.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'refseq': 'XR_934039.3', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}], 'strand': '+', 'symbol': 'MYO15A'}, {'geneid': 124903944, 'is_pseudo': False, 'name': 'uncharacterized LOC124903944', 'rnas': [{'refseq': 'XR_007065652.1', 'so': [{'accession': 'SO:0001627', 'name': 'intron_variant'}]}], 'strand': '-', 'symbol': 'LOC124903944'}]; hg19:{'end': 18054389, 'start': 18054389}; ref:C; rsid:rs765799923; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr17:g.7674149G&gt;A</td>
      <td>1.0</td>
      <td>17</td>
      <td>7674149</td>
      <td>7674149</td>
      <td>True</td>
      <td>alt:A; position:7674149; ref:G;</td>
      <td>N/A</td>
      <td>cdna:{'length': '14955', 'position': '5720'}; cds:{'length': '13284', 'position': '4260'}; effect:synonymous_variant; feature_id:NM_020877.3; feature_type:transcript; gene_id:DNAH2; genename:DNAH2; hgvs_c:c.4260G&gt;A; hgvs_p:p.Lys1420Lys; protein:{'length': '4427', 'position': '1420'}; putative_impact:LOW; rank:27; total:86; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'G', 'freq': {'dbgap_popfreq': 1.0, 'exac': 1.0, 'topmed': 1.0}}, {'allele': 'A', 'freq': {'dbgap_popfreq': 0.0, 'exac': 0.0, 'topmed': 0.0}}]; alt:A; chrom:17; dbsnp_build:156; gene:{'geneid': 146754, 'is_pseudo': False, 'name': 'dynein axonemal heavy chain 2', 'rnas': [{'protein_product': {'refseq': 'NP_001290199.1'}, 'refseq': 'NM_001303270.2', 'so': [{'accession': 'SO:0002152', 'name': 'genic_downstream_transcript_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 5327, 'seq_id': 'NM_020877.5'}, 'hgvs': 'NM_020877.5:c.4260=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1419, 'seq_id': 'NP_065928.2'}}}, 'protein_product': {'refseq': 'NP_065928.2'}, 'refseq': 'NM_020877.5', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_011523663.2'}, 'hgvs': 'XM_011523663.2:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_011521965.1'}}}, 'protein_product': {'refseq': 'XP_011521965.1'}, 'refseq': 'XM_011523663.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4495, 'seq_id': 'XM_011523667.3'}, 'hgvs': 'XM_011523667.3:c.4260=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1419, 'seq_id': 'XP_011521969.1'}}}, 'protein_product': {'refseq': 'XP_011521969.1'}, 'refseq': 'XM_011523667.3', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_011523670.3'}, 'hgvs': 'XM_011523670.3:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_011521972.1'}}}, 'protein_product': {'refseq': 'XP_011521972.1'}, 'refseq': 'XM_011523670.3', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_017024219.2'}, 'hgvs': 'XM_017024219.2:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_016879708.1'}}}, 'protein_product': {'refseq': 'XP_016879708.1'}, 'refseq': 'XM_017024219.2', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4741, 'seq_id': 'XM_047435424.1'}, 'hgvs': 'XM_047435424.1:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_047291380.1'}}}, 'protein_product': {'refseq': 'XP_047291380.1'}, 'refseq': 'XM_047435424.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 4605, 'seq_id': 'XM_047435425.1'}, 'hgvs': 'XM_047435425.1:c.4506=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 1501, 'seq_id': 'XP_047291381.1'}}}, 'protein_product': {'refseq': 'XP_047291381.1'}, 'refseq': 'XM_047435425.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2967, 'seq_id': 'XM_047435426.1'}, 'hgvs': 'XM_047435426.1:c.2898=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 965, 'seq_id': 'XP_047291382.1'}}}, 'protein_product': {'refseq': 'XP_047291382.1'}, 'refseq': 'XM_047435426.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2670, 'seq_id': 'XM_047435427.1'}, 'hgvs': 'XM_047435427.1:c.2511=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 836, 'seq_id': 'XP_047291383.1'}}}, 'protein_product': {'refseq': 'XP_047291383.1'}, 'refseq': 'XM_047435427.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'AAG', 'inserted_sequence': 'AAG', 'position': 2653, 'seq_id': 'XM_047435428.1'}, 'hgvs': 'XM_047435428.1:c.2502=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'K', 'inserted_sequence': 'K', 'position': 833, 'seq_id': 'XP_047291384.1'}}}, 'protein_product': {'refseq': 'XP_047291384.1'}, 'refseq': 'XM_047435428.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}], 'strand': '+', 'symbol': 'DNAH2'}; hg19:{'end': 7674149, 'start': 7674149}; ref:G; rsid:rs751380125; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr12:g.94071094A&gt;C</td>
      <td>1.0</td>
      <td>12</td>
      <td>94071094</td>
      <td>94071094</td>
      <td>True</td>
      <td>alt:C; position:94071094; ref:A;</td>
      <td>N/A</td>
      <td>distance_to_feature:399; effect:upstream_gene_variant; feature_id:NM_001320099.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_001320101.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_003805.4; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NM_001320100.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |distance_to_feature:53; effect:upstream_gene_variant; feature_id:NR_135147.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:n.-53A&gt;C; putative_impact:MODIFIER; transcript_biotype:pseudogene; |distance_to_feature:1451; effect:upstream_gene_variant; feature_id:NM_001330126.1; feature_type:transcript; gene_id:CRADD; genename:CRADD; hgvs_c:c.-1457A&gt;C; putative_impact:MODIFIER; transcript_biotype:protein_coding; |effect:intergenic_region; feature_id:SOCS2-CRADD; feature_type:intergenic_region; gene_id:SOCS2-CRADD; genename:SOCS2-CRADD; hgvs_c:n.94071094A&gt;C; putative_impact:MODIFIER; |</td>
      <td>alleles:[{'allele': 'A', 'freq': {'dbgap_popfreq': 1.0, 'gnomad': 1.0, 'topmed': 1.0}}, {'allele': 'C', 'freq': {'dbgap_popfreq': 0.0, 'gnomad': 0.0, 'topmed': 0.0}}]; alt:C; chrom:12; dbsnp_build:156; gene:{'geneid': 8738, 'is_pseudo': False, 'name': 'CASP2 and RIPK1 domain containing adaptor with death domain', 'rnas': [{'protein_product': {'refseq': 'NP_001307028.1'}, 'refseq': 'NM_001320099.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001307029.1'}, 'refseq': 'NM_001320100.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001307030.1'}, 'refseq': 'NM_001320101.3', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_001317055.1'}, 'refseq': 'NM_001330126.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'NP_003796.1'}, 'refseq': 'NM_003805.5', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'refseq': 'NR_135147.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_016875634.1'}, 'refseq': 'XM_017020145.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_024305020.1'}, 'refseq': 'XM_024449252.2', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_047285760.1'}, 'refseq': 'XM_047429804.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}, {'protein_product': {'refseq': 'XP_047285761.1'}, 'refseq': 'XM_047429805.1', 'so': [{'accession': 'SO:0001986', 'name': 'upstream_transcript_variant'}]}], 'so': [{'accession': 'SO:0001636', 'name': '2KB_upstream_variant'}], 'strand': '+', 'symbol': 'CRADD'}; hg19:{'end': 94071094, 'start': 94071094}; ref:A; rsid:rs1173928494; vartype:snv;</td>
      <td>+</td>
    </tr>
    <tr>
      <td>chr5:g.453975C&gt;T</td>
      <td>1.0</td>
      <td>5</td>
      <td>453975</td>
      <td>453975</td>
      <td>True</td>
      <td>alt:T; position:453975; ref:C;</td>
      <td>N/A</td>
      <td>cdna:{'length': '2762', 'position': '983'}; cds:{'length': '2238', 'position': '855'}; effect:synonymous_variant; feature_id:NM_007277.4; feature_type:transcript; gene_id:EXOC3; genename:EXOC3; hgvs_c:c.855C&gt;T; hgvs_p:p.Tyr285Tyr; protein:{'length': '745', 'position': '285'}; putative_impact:LOW; rank:4; total:13; transcript_biotype:protein_coding;</td>
      <td>alleles:[{'allele': 'C', 'freq': {'dbgap_popfreq': 1.0, 'exac': 1.0, 'gnomad': 1.0, 'gnomad_exomes': 1.0, 'goesp': 1.0, 'mgp': 0.998, 'topmed': 1.0}}, {'allele': 'T', 'freq': {'dbgap_popfreq': 0.0, 'exac': 0.0, 'gnomad': 0.0, 'gnomad_exomes': 0.0, 'goesp': 0.0, 'mgp': 0.002, 'topmed': 0.0}}]; alt:T; chrom:5; dbsnp_build:156; gene:{'geneid': 11336, 'is_pseudo': False, 'name': 'exocyst complex component 3', 'rnas': [{'codon_aligned_transcript_change': {'deleted_sequence': 'TAC', 'inserted_sequence': 'TAC', 'position': 1023, 'seq_id': 'NM_007277.5'}, 'hgvs': 'NM_007277.5:c.855=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'Y', 'inserted_sequence': 'Y', 'position': 284, 'seq_id': 'NP_009208.2'}}}, 'protein_product': {'refseq': 'NP_009208.2'}, 'refseq': 'NM_007277.5', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}, {'codon_aligned_transcript_change': {'deleted_sequence': 'TAC', 'inserted_sequence': 'TAC', 'position': 1023, 'seq_id': 'XM_047416683.1'}, 'hgvs': 'XM_047416683.1:c.855=', 'protein': {'variant': {'spdi': {'deleted_sequence': 'Y', 'inserted_sequence': 'Y', 'position': 284, 'seq_id': 'XP_047272639.1'}}}, 'protein_product': {'refseq': 'XP_047272639.1'}, 'refseq': 'XM_047416683.1', 'so': [{'accession': 'SO:0001580', 'name': 'coding_sequence_variant'}]}], 'strand': '+', 'symbol': 'EXOC3'}; hg19:{'end': 453975, 'start': 453975}; ref:C; rsid:rs376240114; vartype:snv;</td>
      <td>+</td>
    </tr>
  </tbody>
</table></div>

    <script>
        $(document).ready(function() {{
            $('table.display').DataTable({{
                paging: true,
                searching: true,
                ordering: true,
                info: true
            }});
        }});
    </script>

    <audio autoplay loop>
        <source src="magiczna_muzyka.mp3" type="audio/mpeg">
    </audio>

    <!-- <script>
        function spawnRandomImage() {{
            const img = document.createElement('img');
            img.src = 'phd.png';
            img.style.position = 'fixed';
            img.style.width = '100px'; img.style.opacity = '0';
            const x = Math.random()*(window.innerWidth-100);
            const y = Math.random()*(window.innerHeight-100);
            img.style.left = x+'px'; img.style.top = y+'px';
            document.body.appendChild(img);
            setTimeout(() => img.style.opacity='1', 100);
            setTimeout(() => img.style.opacity='0', 3000);
            setTimeout(() => img.remove(), 3000);
        }}
        setInterval(spawnRandomImage, 3000);
    </script>

    <script>
        const mucha = document.createElement("img");
        mucha.src = "mucha.png";
        mucha.style.position = "fixed";
        mucha.style.width = "80px";
        mucha.style.zIndex = 9999;
        mucha.style.top = "50%";
        mucha.style.left = "50%";
        mucha.style.transition = "top 1s linear, left 1s linear";
        document.body.appendChild(mucha);

        function moveFly() {{
            const maxX = window.innerWidth - 80;
            const maxY = window.innerHeight - 80;
            const x = Math.random() * maxX;
            const y = Math.random() * maxY;
            mucha.style.left = x + "px";
            mucha.style.top = y + "px";
        }}

        setInterval(moveFly, 1000);
    </script> -->
</body>
</html>
